app/src/main/java/com/github/se/signify/model/stats/StatsViewModel.kt

```diff
@@ -12,6 +12,25 @@ class StatsViewModel(
     private val repository: StatsRepository,
 ) : ViewModel() {
 
+  sealed class UpdateStatsEvent {
```

Comment:
Could use `data object` instead of `object` inside the sealed class.

Reply:
Thanks, done